python/run-tests

```diff
@@ -57,54 +57,54 @@ function run_test() {
 
 function run_core_tests() {
     echo "Run core tests ..."
-    run_test "pyspark/rdd.py"
-    run_test "pyspark/context.py"
-    run_test "pyspark/conf.py"
-    PYSPARK_DOC_TEST=1 run_test "pyspark/broadcast.py"
```

Comment:
I removed PYSPARK_DOC_TEST to make the doctests consistent with the other pyspark tests since I was changing all of the other test names to modules instead of filepaths.

The two doctests now have the appropriate main function which runs the doctests for that module.